src/App.tsx

```diff
@@ -1,34 +1,63 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import './App.scss';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { FilterBy } from './types/FilterBy';
 
 export const App: React.FC = () => {
+  const todoByDefault: Todo = {
+    userId: 0,
+    id: 0,
+    title: '',
+    completed: false,
+  };
+
+  const [todo, setTodo] = useState(todoByDefault);
```

Comment:
in this case, to make your code more flexible I wouldn't use the default todo but the null
also, I would change the naming a bit
`const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);`
in case of any issues with the id of the user or if users ids start with 0 you can have unwanting errors